crates/syntax/src/test_programs.rs

```diff
@@ -0,0 +1,135 @@
+pub const PARSER_TEST_1: &str = r#"pragma circom 2.0.0;
+
```

Comment:
This only use for testing so we don't need make it public. Just move to the test files.